solver/src/driver.rs

```diff
@@ -80,13 +71,21 @@ impl Driver {
 
     async fn collect_estimated_prices(
         &self,
-        limit_orders: &[LimitOrder],
+        liquidity: &[Liquidity],
     ) -> HashMap<H160, BigRational> {
         // Computes set of traded tokens (limit orders only).
-        let tokens: Vec<H160> = limit_orders
+        let tokens: Vec<H160> = liquidity
             .iter()
             // .flat_map(|lo| vec![lo.sell_token, lo.buy_token].into_iter())
```

Comment:
Probably don't need this anymore. Unless its a hint at whats going on below in "plain rust"